pages/docs/adapters/calculix/adapter-calculix-get-calculix.md

```diff
@@ -10,7 +10,7 @@ The CalculiX adapter for preCICE directly modifies the source code of CalculiX a
 [CalculiX](http://www.dhondt.de) consists of the solver, called "CCX" and a pre- and postprocessing software with graphical user interface "CGX".
 
 - The installation procedure of CCX is described in its `src/README.INSTALL` files, but we also give a summary here.
-- We don't modify CGX, so you can simply get a binary package (if needed, e.g. as a preprocessor in our FSI tutorials)
+- We don't modify CGX, so you can get a binary package (if needed, e.g. as a preprocessor in our FSI tutorials)
```

Comment:
Here another term would be helpful to emphasize that a binary package is the simpler approach compared to building from source. Maybe "directly"?

Reply:
Why not simply:

> The adapter is fully compativle with CGX. You can use it from your package repository or compile it from source.

Reply:
Ah right, that was about CGX. It is simply fine both as you changed it, as well as you suggested in your last comment.